dcat/rdf/dcat.ttl

```diff
@@ -151,9 +151,9 @@
 dcat:Catalog
   a rdfs:Class ;
   a owl:Class ;
-  rdfs:comment "A curated collection of metadata about datasets and data services"@en ;
-  rdfs:comment "Una colección seleccionada de metadatos sobre conjuntos de datos o servicios de datos"@es ;
-  rdfs:comment "Una raccolta curata di metadati sui dataset e relativi servizi"@it ;
+  rdfs:comment "A curated collection of metadata about cataloged resources (e.g., datasets and data services in the context of a data catalog)."@en ;
+  rdfs:comment "Una colección curada de metadatos sobre recursos catalogados (por ejemplo, conjuntos de datos y servicios de datos en el contexto de un catálogo de datos)."@es ;
```

Comment:
@agbeltran Could you doublecheck the Spanish translation?

Reply:
@riccardoAlbertoni the Spanish translation is good!